Dockerfile

```diff
@@ -157,5 +157,15 @@ RUN apt-get update -qq && apt-get -y --no-install-recommends install \
 RUN R -e "install.packages('DT', dependencies = TRUE)"
 RUN R -e "BiocManager::install(c('rtracklayer'), update = FALSE)"
 
+# Install python3 data science basics (pandas)
+# using pip to get more current versions
+RUN apt-get update -qq && apt-get -y --no-install-recommends install python3-pip 
+RUN pip3 install "numpy==1.17.3" && \
+   pip3 install "six==1.13.0" "setuptools==41.6.0" && \
+   pip3 install "cycler==0.10.0" "kiwisolver==1.1.0" "pyparsing==2.4.5" "python-dateutil==2.8.1" "pytz==2019.3" && \
+   pip3 install "matplotlib==3.0.3" && \
+   pip3 install "scipy==1.3.2" && \
+   pip3 install "pandas==0.25.3"
+
```

Comment:
Did you want to set the default python to python3 or should we remind folks to use python3 in their shell scripts/CI sections if they want to run with it?

Reply:
Actually - now that I'm typing this out I think just reminding folks to use the `python3` command is the way to go.

Reply:
I decided not to change it; I was worried about breaking other things in strange ways.